Documentation/Index.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: Includes.txt
+
+.. _start:
+
+=============================================================
+EXT:tea
+=============================================================
+
+:Version:
+   |release|
+
+:Language:
+   en
+
+:Description:
+   This TYPO3 extension is an example of best practices in continuous integration and automated code checks, also
+   Extbase/Fluid-based extensions for TYPO3 CMS using PHPUnit.	writing unit and functional tests for Extbase/Fluid-based extensions for TYPO3 CMS using PHPUnit.
```

Comment:
There should only be one space after the comma.

Also, lines should be no longer than 80 characters.